### PR TITLE
Add footer

### PR DESCRIPTION
**Pull request by GNUp**

### CODE REVIEW

**Comment by GNUp** [comment]
@foriequal0 Please review this PR

File: src/Footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+                <span className="link-header">Blog</span>
+              </a>
+            </div>
+          </div>
+        </Container>
+      </div>
+    );
+  }
+
+  private linkHandler = (id: string) => () => {
```
Comment (foriequal0) [inline src/Footer/Footer.tsx]: This doesn't seem to be used.

**Comment by sgkim126** [comment]
@GNUp When the UI changes, please upload screenshots comparing before and after the modification.

**Comment by GNUp** [comment]
__Before__
There was no footer
__After__
image
image

image

**Comment by foriequal0** [comment]
I think those bugs can be fixed later. Can you check that the function is safe to remove? Then I'll approve this.

File: src/Footer/Footer.tsx (hunk 1 above)
Comment (GNUp) [reply src/Footer/Footer.tsx]: I removed it

**Comment by GNUp** [comment]
* I removed the function
* I removed the gap at the bottom

**Pull request merged**